src/inmanta/deploy/state.py

```diff
@@ -329,6 +466,9 @@ def update_requires(self, resource: ResourceIdStr, requires: Set[ResourceIdStr])
         """
         Update the requires relation for a resource. Updates the reverse relation accordingly.
         """
+
+        # TODO: verify this is correct and needed
```

Comment:
@sanderr I don't know about this bit very well.

Reply:
I don't think this is needed anymore. But you also have to check for `BlockedStatus.TRANSIENT`  when unblocking.